src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,10 +10,11 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_COMPANY = new Prefix("com/");
```

Comment:
standardise the format of the prefix for deliveries and suppliers, decide whether / should come at the front or the back